### PR TITLE
Remove extra white space at top of wizard.

### PR DESCRIPTION
**Pull request by JoshDunnDev**

### CODE REVIEW

File: FinishButton.vue [hunk 1]
```diff
@@ -29,6 +35,7 @@ export default {
   },
   methods: {
     finish() {
+      this.disabled = true;
       this.$emit('finish');
```
Comment (dlongley) [inline FinishButton.vue]: We're already addressing the disabled/blocked issue in PR #8. Let's get that PR fixed up, reviewed again, and merged and remove these changes here. @gannan08 -- that PR needs your attention for changes and then I can review again and merge.

**Review by dlongley — CHANGES_REQUESTED** [review]
We need to focus this PR on the spacing and get PR #8 finished and merged -- see comment.

**Comment by gannan08** [comment]
@JoshDunnDev Can we rebase this on top of master and squash some of these commits. We have 12 commits for 10 line code changes across two files.

**Comment by JoshDunnDev** [comment]
> @JoshDunnDev Can we rebase this on top of master and squash some of these commits. We have 12 commits for 10 line code changes across two files.

There is really only 3 commits that I did. I think extra commits got added when it was previously rebased.

**Pull request closed without merging**